main.ts

```diff
@@ -79,7 +85,9 @@ const results: [fileName: string, exitCode: number][] = []
 const runner = browser
   ? await runWithBrowser({ createBrowser, importMapUrl, results })
   : await runWithDeno({ reloadUrl: flags.reload, signal, results })
-const paths = sourceFileNames.map((fileName) => [dir, fileName] as const)
+
+console.log(`${paths.length} files found`)
+console.log(paths)
```

Comment:
Perhaps just `testing ${paths.length} scripts`? Logging out the `paths` array may or may not be useful. Thoughts?

Reply:
I think its useful to log out the paths because were dealing with globs that include and exclude files, so it is a nice sanity check that all the files we want to target are being hit.